packages/discussions/lib/urlClipper.js

```diff
@@ -0,0 +1,16 @@
+const clipUrlInText = (url, content) => {
```

Comment:
I've some cute naming suggestions:

- `stripUrl`
- `stripUrlInContent` (or `stripUrlInText` if argument is named `text` instead of content
- `stripLonelyUrl`

Reply:
re naming: clip or strip is not such a big difference and there is already `nameClipper` in the discussions code base.
re `stripLonelyUrl`: this piece is in the context of `discussion` and the logic might change, that's why I want to keep the name general.